part-1/2.html

```diff
@@ -42,12 +42,13 @@
   <div id="text">
     <p>Welcome to the Land of Bump! Everything here is very round. There are round ponds, round trees and round houses.</p>
   </div>
-  <a href="./2.html" id="audio" title="Play audio">play audio</a>
+  <a href="./2.html#" id="audio" title="Play audio">play audio</a>
```

Comment:
Seeing this change makes me realize something I missed before.

You probably know this, and I should've known better when I put the original pages together, but any `a` element whose `href` value is simply `#` in the original HTML really should've been a `button` element instead.

The current code will function OK, though, and there's probably not time to spend changing it everywhere, updating the CSS, etc. So it's fine if you leave it as is. I just wanted to make sure to mention it for posterity.